spec/presenters/tree_builder_alert_profile_obj_spec.rb

```diff
@@ -0,0 +1,126 @@
+describe TreeBuilderAlertProfileObj do
+  before do
+    role = MiqUserRole.find_by(:name => "EvmRole-operator")
+    group = FactoryGirl.create(:miq_group, :miq_user_role => role, :description => "Tags Group")
+    login_as FactoryGirl.create(:user, :userid => 'tags_wilma', :miq_groups => [group])
+  end
+
+  let!(:tag1a) { FactoryGirl.create(:classification, :name => 'tag1a') }
+  let!(:tag2a) { FactoryGirl.create(:classification, :name => 'tag2a') }
+  let!(:tag3a) { FactoryGirl.create(:classification, :name => 'tag3a') }
+  let!(:folder1a) do
+    f1 = FactoryGirl.create(:classification, :name => 'folder1a', :show => true)
+    f1.entries.push(tag1a)
+    f1.entries.push(tag2a)
+    f1.entries.push(tag3a)
+    f1
+  end
+  let!(:tag1b) { FactoryGirl.create(:tenant, :name => 'tag1b') }
+  let!(:tag2b) { FactoryGirl.create(:tenant, :name => 'tag2b') }
+  let!(:tag3b) { FactoryGirl.create(:tenant, :name => 'tag3b') }
+  let!(:tree_name) { :alert_profile_obj }
```

Comment:
I see that this chat does not auto refresh the messages...i noticed the messages in the mail and then i had to refresh...so if Im like stupid for you, it is because i did not saw your messages :-)